src/components/model.js

```diff
@@ -90,4 +99,7 @@ module.exports.Component = registerComponent('loader', {
       el.setObject3D('mesh', collada.scene);
     });
   }
-});
+};
+
+module.exports.Component = registerComponent('loader', ModelComponent);
+module.exports.Component = registerComponent('model', ModelComponent);
```

Comment:
I really don't want to deal with this right now. Old scenes will depend on the old version of aframe so they will be fine.